services/gitea/8.2.0/defaults/cm.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 data:
   values.yaml: |-
     ---
+    priorityClassName: "dkp-critical-priority"
```

Comment:
for gitea, we'll need to apply the priority class to postgres/memcached as well

```
priorityClassName: <priority class>

memcached:
  priorityClassName: <priority class>

postgresql:
  primary:
    priorityClassName: <priority class>
```

Reply:
great catch!! done